### DISCUSSION TITLE
Warning: File 'repomd.xml' from repository 'Puppet 7 Repository sles 12 - x86_64' is signed with an unknown key '4528B6CD9E61EF26'

### DISCUSSION THREAD

**Discussion by deepakpal-in** [puppetlabs/puppet-agent#2168] [General]
Hello Team,

While trying to install the puppet agent on the SLES 12.5 I am facing difficulty of unknown key as mentioned in the subject.

Due to this issue my auto-installer on AWS image builder is also rejecting the package. To follow the strict compliance I don't want to install anything unverified in my environment.

Can you please by adding a correct key to the packages or let me know how we can add a correct key to the package. 

Please see below warning messages for more info: 

------------------------------
Refreshing service 'Web_and_Scripting_Module_x86_64'.
Warning: File 'repomd.xml' from repository 'Puppet 7 Repository sles 12 - x86_64' is signed with an unknown key '4528B6CD9E61EF26'.

    Note: Signing data enables the recipient to verify that no modifications occurred after the data
    were signed. Accepting data with no, wrong or unknown signature can lead to a corrupted system
    and in extreme cases even to a system compromise.

    Note: File 'repomd.xml' is the repositories master index file. It ensures the integrity of the
    whole repo.

    Warning: We can't verify that no one meddled with this file, so it might not be trustworthy
    anymore! You should not continue unless you know it's safe.

File 'repomd.xml' from repository 'Puppet 7 Repository sles 12 - x86_64' is signed with an unknown key '4528B6CD9E61EF26'. Continue? [yes/no] (no): no

------------------------------

**Comment by deepakpal-in** [comment]
I am facing same issue with below repo as well:

http://yum.puppetlabs.com/sles/12/PC1/x86_64/puppetlabs-release-pc1-1.1.0-5.sles12.noarch.rpm

**Comment by joshcooper** [comment]
This is likely due to the changes described in this post https://groups.google.com/g/puppet-dev/c/mPsyy1A5YUU/m/dWqUMP3SBQAJ

Since the PC1 repos are EOL, please use these instead:

https://yum.puppet.com/puppet7-release-sles-12.noarch.rpm
https://yum.puppet.com/puppet8-release-sles-12.noarch.rpm
